fbchat/client.py

```diff
@@ -3035,6 +3049,18 @@ def onUserJoinedCall(self, mid=None, joined_id=None, is_video_call=None, thread_
         """
         log.info("{} joined call in {} ({})".format(joined_id, thread_id, thread_type.name))
 
+        
+        
+    def onWaved(self, author_id, thread_id=None, thread_type=None, ts=None):
```

Comment:
For future, `author_id` should be `None`, and by the way you should include `msg` and `metadata` parameters too (like in other methods)